webpack.config.js

```diff
@@ -42,6 +43,40 @@ const alias = {
 		__dirname,
 		'assets/js/settings/blocks'
 	),
+	'@woocommerce/base-components': path.resolve(
+		__dirname,
+		'assets/js/base/components/'
+	),
+	'@woocommerce/base-hocs': path.resolve( __dirname, 'assets/js/base/hocs' ),
+	'@woocommerce/block-components': path.resolve(
+		__dirname,
+		'assets/js/components/'
+	),
+	'@woocommerce/block-hocs': path.resolve( __dirname, 'assets/js/hocs' ),
+};
+
+const legacyAliasResolve = {
+	plugins: [
+		new FallbackModuleDirectoryPlugin( '/legacy/', '/', {
+			...alias,
+			'@woocommerce/base-components': path.resolve(
+				__dirname,
+				'assets/js/legacy/base/components/'
+			),
+			'@woocommerce/base-hocs': path.resolve(
+				__dirname,
+				'assets/js/legacy/base/hocs'
```

Comment:
I'm a bit confused here, why `'assets/js/legacy/base/components/'` ends with a slash while `'assets/js/legacy/base/hocs'` doesn't?

Reply:
It should end with a slash.  I'll get that fixed.  Good catch!

Reply:
see 60c5724 but it got overridden by the webpack config refactor (which preserved it).